deploy/metric-adapter/rbac.yaml

```diff
@@ -3,40 +3,9 @@ kind: ClusterRole
 metadata:
   name: metric-adapter
 rules:
-  - apiGroups:
```

Comment:
Because now the adapter need to list the workload scale to get the current replicas when cron is not active, we do not know what kind group, it will forbidden when not specified